src/main/java/com/hulu/ftl/formats/XMLFormat.java

```diff
@@ -82,7 +80,9 @@ public XMLFormat(InputStream stream) throws IOException {
                         .evaluate(rootNode, XPathConstants.NODESET);
 
                 for (int i = 0; i < nodes.getLength(); i++) {
-                    values.add(nodes.item(i).getNodeValue());
+                    String value = nodes.item(i).getNodeValue();
+                    if (!Pattern.matches("\\s*", value))
```

Comment:
This is skipping any value that's entirely whitespace (because sometimes ./text() claims to have a value, but it's just like a newline or something)